mmv1/third_party/terraform/resources/resource_compute_security_policy.go.erb

```diff
@@ -439,6 +468,21 @@ func resourceComputeSecurityPolicy() *schema.Resource {
 					},
 				},
 			},
+			"recaptcha_options_config": {
+				Type: schema.TypeList,
+				Optional: true,
+				Description: `reCAPTCHA configuration options to be applied for the security policy.`,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"redirect_site_key": {
+							Type: schema.TypeString,
+							Required: true,
```

Comment:
The API mentions that redirectSiteKey is optional field. If not specified, a Google-managed site key is used. I make it required, because it is the only field in the object recaptchaOptionsConfig. Just want to check if it is OK?

Reply:
yes, that's a reasonable thing to do. If it turns out to not be required we can always make it optional later.